.golangci.yml

```diff
@@ -186,6 +186,9 @@ issues:
     - linters:
         - staticcheck
       text: "GetDeprecatedLegacyJsonFieldConflicts is deprecated"
+    - linters:
+        - staticcheck
+      text: "GetIgnoreEmpty is deprecated"
```

Comment:
Would optimally scope this to the specific file (the one above should be too, but doesn't need to be in this PR)

Reply:
Fixed.